x/observer/migrations/v11/migrate.go

```diff
@@ -0,0 +1,50 @@
+package v11
+
+import (
+	sdk "github.com/cosmos/cosmos-sdk/types"
+
+	"github.com/zeta-chain/node/x/observer/types"
+)
+
+type observerKeeper interface {
+	GetBallotListForHeight(ctx sdk.Context, height int64) (val types.BallotListForHeight, found bool)
+	DeleteBallot(ctx sdk.Context, index string)
+	DeleteBallotListForHeight(ctx sdk.Context, height int64)
+	GetAllBallots(ctx sdk.Context) (ballots []*types.Ballot)
+}
+
+const (
+	PendingBallotsDeletionBufferBlocks = int64(144000)
+	MaturityBlocks                     = int64(100)
+)
+
+// MigrateStore migrates the x/observer module state from consensus version 10 to version 11.
+// The migration deletes all ballots and ballot lists for heights less than the maturity blocks.
```

Comment:
Don't we just want to delete ballots at height 0?

Reply:
Yes, but the comment is more accurate for the logic being used